include/crocoddyl_msgs/inertial_parameters_publisher.h

```diff
@@ -0,0 +1,74 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2023-2024, Heriot-Watt University
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_MSG_MULTIBODY_INERTIAL_PARAMETERS_PUBLISHER_H_
+#define CROCODDYL_MSG_MULTIBODY_INERTIAL_PARAMETERS_PUBLISHER_H_
+
+#include <realtime_tools/realtime_publisher.h>
+
+#include <ros/node_handle.h>
+#include "crocoddyl_msgs/MultibodyInertialParameters.h"
+
+typedef crocoddyl_msgs::MultibodyInertialParameters MultibodyInertialParameters;
+
+namespace crocoddyl_msgs {
+
+class MultibodyInertialParametersRosPublisher {
+public:
+  /**
+   * @brief Initialize the multi-body inertial parameters publisher.
+   *
+   * @param[in] n_bodies  Number of bodies
+   * @param[in] topic     Topic name
+   */
+  MultibodyInertialParametersRosPublisher(
+      const unsigned int n_bodies,
+      const std::string &topic = "/robot/multibody_inertial_parameters"):
+      n_bodies_(n_bodies) {
+    ros::NodeHandle n;
+    pub_.init(n, topic, 1);
+    pub_.msg_.parameters.resize(n_bodies_);
```

Comment:
Let's support ROS 2, remove n_bodies, and change the topic name (also add the print message). I suggest similar changes in the subscriber class below.
```suggestion
  /**
   * @brief Initialize the multi-body inertial parameters publisher.
   *
   * @param[in] topic     Topic name
   */
  MultibodyInertialParametersRosPublisher(
      const std::string &topic = "/crocoddyl/inertial_parameters")
#ifdef ROS2
      : node_("inertial_parameters_publisher"),
        pub_(node_.create_publisher<MultibodyInertialParameters>(topic, 1)) {
    RCLCPP_INFO_STREAM(node_.get_logger(),
                       "Publishing MultibodyInertialParameters messages on "
                           << topic);
#else
      : {
    ros::NodeHandle n;
    pub_.init(n, topic, 1);
    ROS_INFO_STREAM("Publishing MultibodyInertialParameters messages on "
                    << topic);
#endif
```